api/src/test/resources/org/openmrs/module/fhir2/api/dao/impl/FhirDiagnosticReportDaoImplTest_initial_data.xml

```diff
@@ -18,5 +18,5 @@
 	<concept_name concept_id="5085" name="FHIR Diagnostic Service Section Codes" locale="en_GB" creator="1" date_created="2004-08-12 00:00:00.0" concept_name_id="1439" concept_name_type="FULLY_SPECIFIED" locale_preferred="1" voided="false" uuid="67340528-feea-4232-b98c-b443c6affb39"/>
 	<obs obs_id="17" person_id="7" concept_id="5085" encounter_id="3" obs_datetime="2008-07-01 00:00:00.0" location_id="1" value_numeric="[NULL]" comments="" creator="1" date_created="2008-08-18 14:09:35.0" voided="false" value_coded="[NULL]" value_coded_name_id="[NULL]" value_complex="[NULL]" value_text="LAB" value_datetime="[NULL]" value_drug="[NULL]" uuid="d899333c-5bd4-45cc-b1e7-2f9542dbcbf6"/>
 	<obs obs_id="18" person_id="7" concept_id="5085" encounter_id="3" obs_datetime="2008-07-01 00:00:00.0" obs_group_id="17" location_id="1" value_numeric="66" comments="" creator="1" date_created="2008-08-18 14:09:35.0" voided="false" value_coded="[NULL]" value_coded_name_id="[NULL]" value_complex="[NULL]" value_text="LAB" value_datetime="[NULL]" value_drug="[NULL]" uuid="1e589127-f391-4d0c-8e98-e0a158b2be22"/>
-	<obs obs_id="19" person_id="7" concept_id="5085" encounter_id="3" obs_datetime="2008-07-01 00:00:00.0" location_id="1" value_numeric="[NULL]" comments="" creator="1" date_created="2008-08-18 14:09:35.0" voided="false" value_coded="[NULL]" value_coded_name_id="[NULL]" value_complex="[NULL]" value_text="LAB" value_datetime="[NULL]" value_drug="[NULL]" uuid="dc386962-1c42-49ea-bed2-97650c66f742"/>
+	<obs obs_id="19" person_id="8" concept_id="5085" encounter_id="3" obs_datetime="2008-07-01 00:00:00.0" location_id="1" value_numeric="[NULL]" comments="" creator="1" date_created="2008-08-18 14:09:35.0" voided="false" value_coded="[NULL]" value_coded_name_id="[NULL]" value_complex="[NULL]" value_text="LAB" value_datetime="[NULL]" value_drug="[NULL]" uuid="dc386962-1c42-49ea-bed2-97650c66f742"/>
```

Comment:
Why did you change the person associated with this obs?

Reply:
It becomes difficult to test if all the data is similar. But I could revert if you want.

Reply:
That's reasonable. Leave it as is.

Reply:
@varung-31 another option would be to add more test data if the existing data doesn't suite your test cases

Reply:
@varung-31 I agree with @jecihjoy's suggestion here, that it would be better to add some new data rather than deleting existing data.

Reply:
Done